### PR TITLE
Docs/2023 09 28 updates and fixes

### PR DESCRIPTION
**Pull request by BaxterEaves**
- Fixed some broken links the the book
- Fixed non-working code in the README (used old pylace conventions)
- Added back the `light` theme to book since some windows users were experiencing low contrast with the `latte` theme
- Added some example use cases to the README
- Added output figures to the README

### CODE REVIEW

**Review by schmidmt — APPROVED** [review]
Much better!

File: README.md [hunk 1]
```diff
@@ -105,36 +125,62 @@ themselves from scratch, meaning they must know (or at least guess) the model.
...
 
-### Who should not use lace
+### Example use cases
+
+- **Combine data sources and understand how they interact.** For example, we
+    may wish to predict cognitive decline from demographics, survey or task
+    performance, EKG data, and other clinical data. Combined, this data would
+    typically be very sparse (most patients will not have all fields filled
+    in), and it is difficult to know how to explicitly model the interaction of
+    these data layers. In Lace, we would just concatenate the layers and run
```
Comment (Swandog) [inline README.md:135]: "Concatenate the layers and pull them through"-i don't understand this, it sounds too simple-like how could it be that easy? Maybe a rewording that explains why you can't do that yourself? Or maybe I'm not getting it

File: README.md [hunk 2]
```diff
@@ -105,36 +125,62 @@ themselves from scratch, meaning they must know (or at least guess) the model.
...
 
+
 ## Quick start
 
-Install the CLI and pylace (requires [rust and
-cargo](https://www.rust-lang.org/tools/install))
+### Installation
 
-```console
+Lace requires rust.
```
Comment (Swandog) [inline README.md:164]: `pylace` shouldn't require Rust if you install it with pip

**Review by Swandog — APPROVED** [review]
Had some thoughts, but not enough to hold approval. LGTM!

File: README.md (hunk 1 above)
Comment (BaxterEaves) [reply README.md:135]: It is that easy 😉

**Pull request merged**